CONTRIBUTING.md

```diff
@@ -121,8 +121,8 @@ Note that tests will fail automatically if they do not finish execution within 6
 
 #### STDOUT/STDERR and logging capture
 
-When running tests in parallel using `stestr` either via tox, the Makefile (`make
-test_ci`), or in CI, we set the env variable `QISKIT_TEST_CAPTURE_STREAMS`, which will
+When running tests in parallel using `stestr` either via tox
+or in CI, we set the env variable `QISKIT_TEST_CAPTURE_STREAMS`, which will
```

Comment:
This also needs to be added to `passenv`, right?

Reply:
Actually, the text says that we are setting it, so I added it to `setenv`. This matches what is in the Qiskit tox.ini. We have gone the whole time without it. Funny, I haven't really noticed the difference enough to be conscious of it. Maybe the experiments test runs are a little spammier than the Qiskit ones...